frontend/resources/locales.json

```diff
@@ -1269,13 +1263,13 @@
   "modals.delete-project-confirm.message" : {
     "used-in" : [ "src/app/main/ui/dashboard/files.cljs:57" ],
     "translations" : {
-      "en" : "Are you sure you wan't to delete this project?"
+      "en" : "Are you sure you want to delete this project?"
     }
   },
   "modals.delete-project-confirm.title" : {
     "used-in" : [ "src/app/main/ui/dashboard/files.cljs:56" ],
     "translations" : {
-      "en" : "Deleting project"
+      "en" : "Delete project"
```

Comment:
Why this change? The design/prototypes expectes the `Deleting project` title.

Reply:
It's asked for in this issue https://tree.taiga.io/project/uxboxproject/issue/759

Reply:
I think, they should first agree before fill issues, the prototypes says "Deleting projects". But is ok.